flow/genericactors.actor.h

```diff
@@ -311,7 +311,7 @@ Future<U> mapAsync(Future<T> what, F actorFunc)
 template<class T, class F>
 std::vector<Future<decltype(actorFunc(T()).getValue())>> mapAsync(std::vector<Future<T>> const& what, F const& actorFunc)
 {
-	std::vector<typename std::result_of<F(T)>::type> ret;
+	std::vector<typename std::invoke_result<F(T)>::type> ret;
```

Comment:
Is this the correct usage of `invoke_result`? The examples I've seen show a switch from `F(T)` to something like `F, T`.